packages/api/src/hooks/useTradingPlatformAvailableAccounts.ts

```diff
@@ -1,10 +1,13 @@
 import { useMemo } from 'react';
+import useAuthorize from './useAuthorize';
 import useFetch from '../useFetch';
 
 /** @description This hook is used to get all the available MT5 accounts. */
 const useTradingPlatformAvailableAccounts = () => {
+    const { isSuccess } = useAuthorize();
     const { data: mt5_available_accounts, ...rest } = useFetch('trading_platform_available_accounts', {
         payload: { platform: 'mt5' },
+        options: { enabled: isSuccess },
```

Comment:
@hamza-deriv we don't want wait for authorize in the data layer, this should be handled in the UI part so by right when we call this hook in the UI, the authorize should be there

Reply:
removed